src/components/content/conditions/helpers/history.tsx

```diff
@@ -15,9 +17,35 @@ export function useConditionHistory(condition: ConditionModel) {
     valuesToDedupeOn,
     getSearchParams,
     getHistoryEntry,
+    postQueryFilter: filterConditionResultsPostQuery,
   });
 }
 
+function isChronicConditionLens(system?: string, code?: string) {
+  return system === SYSTEM_ZUS_LENS && code === "ChronicConditions"
+}
+
+function isSummaryConditionLens(system?: string, code?: string) {
+  return system === SYSTEM_SUMMARY && code === "Common"
+}
+
+function filterConditionResultsPostQuery(c: Condition) {
+
+  const metaTags = c.meta?.tag
+  if (!metaTags) {
+    return true;
+  }
+
+  const lenTags = metaTags.filter((tag) => 
+    isChronicConditionLens(tag.system, tag.code) ||
+    isSummaryConditionLens(tag.system, tag.code)
+  )
+
+  const hasLensTags = lenTags.length > 0
```

Comment:
```suggestion
  const lenTags = metaTags.filter((tag) => 
    isChronicConditionLens(tag.system, tag.code) ||
    isSummaryConditionLens(tag.system, tag.code)
  ).length > 0
```

Nit: you can combine the .length and then just do the not of that and return it

Reply:
yeah, I went with your suggestion but also like decoupling what's returned from the business logic. If we wanted to do additional filtering, we'd just have to add another boolean to the return value. Certainly YAGNI though so game for taking the suggestion